src/lib/components/input/input.js

```diff
@@ -27,6 +27,8 @@ const Input = ({
   validationMessage,
   handleValidations,
   isValidationOk,
+  hasErrorAfterSubmit,
+  getValidationMessage,
```

Comment:
Why are you adding this here? Validations was already working with the input,,

Reply:
No necessity to put this here if we are not using it

Reply:
Yes there is – by destructuring it here, we remove it from the props passed to the HTML element. Otherwise, weird properties like "hasErrorAfterSubmit" get passed to the `<input>`

Reply:
ahhh good point @rin !